.gitignore

```diff
@@ -130,3 +130,6 @@ cobertura.xml
 # Python files for UITests
 *.pyc
 /.idea
+
+# The folder bundle puts its installation locally
```

Comment:
I don't understand this?

Reply:
Mhm. My bad I guess. I have it set up so bundler installs all gems in `vendor` directory of your project so it's easy to package up the application for deployment.

Reply:
Ah, makes sense now :) There's probably a way in git of having device-specific git config, maybe?